src/module/chat/chatrollhelpers.ts

```diff
@@ -416,7 +417,7 @@ export async function rollAndDisplayOracleResult(
   }
 
   // Do the random roll
-  const tableDraw = await (table as any).draw({ displayChat: false })
+  const tableDraw = await(table).draw({ displayChat: false } as RollTable.DrawOptions)
```

Comment:
Well that's an unfortunate typing. Too bad this call doesn't accept a `Partial<RollTable.DrawOptions>`.

Reply:
ugh, yeah, that one made me grind my teeth. [according to the API docs](https://foundryvtt.com/api/RollTable.html), the props on `RollTable.DrawOptions` **are** all optional (and, i mean, the rolls work), so i'm guessing this is just an error in the LoFD typings package. TBH, i'm surprised that they're not just extracting them directly from FVTT's source to avoid that kind of error, but maybe there's a good reason not to do that.

either way, i'll add a note about it... and i should probably open a PR there, too.

Reply:
OK, should be good to merge now 👍